src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -734,4 +734,13 @@
   <data name="GeneratorTotalExecutionTime" xml:space="preserve">
     <value>Total generator execution time: {0} seconds.</value>
   </data>
-</root>
+  <data name="Illegal_built_in_operator_name_0" xml:space="preserve">
```

Comment:
Please follow compiler conventions on resource names.

Reply:
Fwiw, I cannot figure out what your convention is.  There seems to be absolutely no relation between the identifiers and the content. :-/

Reply:
All three of these actually follow a convention that keeps things in sync and make the consumption side clear and trivial to verify. I would prefer to keep as is.

Reply:
But it's inconsistent with all the rest of our code. I would suggest `IllegalBuiltInOperatorName` for this one.